docs/securelaunch/securelaunch.policy

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Would recommend naming the file something like `policy.example` for people to understand it is an example